tests/test_community_fault_model.py

```diff
@@ -0,0 +1,24 @@
+from source_modelling import community_fault_model
+from source_modelling.community_fault_model import NodalPlane
+import numpy as np
+import pandas as pd
+
+
+
+def test_can_load_community_fault_model():
+    model = community_fault_model.get_community_fault_model()
+    assert len(model) == 880
+    gdf = community_fault_model.community_fault_model_as_geodataframe()
+    assert len(gdf) == 880
+
+def test_most_likely_nodal_plane():
+    solutions = pd.read_csv('tests/data/GeoNet_Test_Solutions.csv')
+    model = community_fault_model.get_community_fault_model()
+    correct = 0
+    for _, solution in solutions.iterrows():
+        nodal_plane_1 = NodalPlane(solution['strike1'], solution['dip1'], solution['rake1'])
+        nodal_plane_2 = NodalPlane(solution['strike2'], solution['dip2'], solution['rake2'])
+        if community_fault_model.most_likely_nodal_plane(model, np.array([solution['Latitude'], solution['Longitude']]), nodal_plane_1, nodal_plane_2) == nodal_plane_1:
+            correct += 1
+
+    assert correct > 75
```

Comment:
75 because you already know? is "greater than" a good test?. Can `correct` be 76 and 77 and both are ok?

Reply:
it's pretty arbitrary. It's just that *if* I introduced a change to the code in the future that lowered the accuracy to say 50% that test would let me know something is wrong.

Reply:
If, however accuracy improves or goes down by one or two I don't really care.